src/frontend/app/shared/components/list/list-cards/cards.component.ts

```diff
@@ -1,14 +1,15 @@
 import { Component, Input } from '@angular/core';
 
 import { IListDataSource } from '../data-sources-controllers/list-data-source-types';
-import { TableCellCustom } from '../list.types';
+import { CardCell } from '../list.types';
 
 @Component({
   selector: 'app-cards',
   templateUrl: './cards.component.html',
   styleUrls: ['./cards.component.scss']
 })
 export class CardsComponent<T> {
+  public columns = CardCell.columns;
```

Comment:
This breaks `CfEndpointCardComponent` which overrides the static `component.columns`/`CardCell.columns`